cmd/tailscale/ui.go

```diff
@@ -260,7 +265,9 @@ func (ui *UI) layout(gtx layout.Context, sysIns system.Insets, state *clientStat
 
 	for _, e := range ui.search.Events() {
 		if _, ok := e.(widget.ChangeEvent); ok {
-			events = append(events, SearchEvent{Query: ui.search.Text()})
+			text := ui.search.Text()
+			ui.showDebugMenu = strings.EqualFold(text, "debug")
```

Comment:
I think this means that if the debug menu is enabled, searching for anything which isn't "debug" will disable the menu.

If that is the intent that is fine, this is clearly a transitory mechanism. Otherwise, it could be conditional to only set not clear ui.showDebugMenu.

Reply:
oh, that's an interesting idea, to make it a one-time enable and not go away once you've entered the magic code.

Reply:
Done.